src/styles/base.scss

```diff
@@ -1484,8 +1485,20 @@ td, th {
   line-height: 40px;
 }
 
+/*pagination*/
+
 .btn-pagination {
-  background-color: $yellow;
+  background-color: $highlight;
+  i {
+    font-size: 16px;
+  }
+}
+
+.btn-pagination:hover {
```

Comment:
This is not creating an effect. As styles with and without **: hover** are same.